scripts/build.ps1

```diff
@@ -23,41 +24,41 @@ Push-Location (Join-Path $workpath "v8build\v8")
 # Generate the build system
 $gnargs = 'v8_enable_i18n_support=false is_component_build=false v8_monolithic=true v8_use_external_startup_data=false treat_warnings_as_errors=false'
 
-# commenting this out for now since Android build is at android/build.sh
-
-# if ($Configuration -like "*android") {
-#     $gnargs += ' use_goma=false target_os=\"android\" target_cpu=\"' + $Platform + '\"'
-# }
-# else {
-if (-not ($UseLibCpp)) {
-    $gnargs += ' use_custom_libcxx=false'
-}
+if ("android linux mac".contains($AppPlatform)) {
+    $gnargs += ' v8jsi_enable_napi=false use_goma=false target_os=\"' + $AppPlatform + '\"'

```

Comment:
>v8jsi_enable_napi=false [](http://example.com/codeflow?start=17&length=23)

Curious, are there any big issues with enabling current napi implementation on non-windows ?

Reply:
I had more errors when I first started, but just tried it now and it doesn't look so bad. I think I'll just fix those and just get rid of this build variable.